src/mono/mono/metadata/loader-internals.h

```diff
@@ -98,10 +97,15 @@ struct _MonoMemoryManager {
 	// Currently unused, we take the domain lock instead
```

Comment:
This comment is wrong now.

```suggestion
	// Must be acquired after the loader lock
```